makeservices/easywp

```diff
@@ -57,7 +57,7 @@ fi
 
 # Get SQL password
 echo "Resetting database password..."
-sqlpass=$(echo "yes" | makemysql | tail -n 1 | grep -Po '(?<=: )([0-9a-zA-Z]){24,}$')
+sqlpass=$(makemysql --unreadable-output --silent)
```

Comment:
seems like you should have `--ignore-wp` here too, since there won't be any wordpress installation yet?

Reply:
`--ignore-wp` is for those who installed WordPress but don't want to change their pw, here the thing will be done by `wp-cli` in `makemysql` imo